### PR TITLE
PR to update Partitions page

### PR DESCRIPTION
**Pull request by manasvita**
Added a para about job violates account/QOS policy in the Notes section

### CODE REVIEW

**Comment by manasvita** [comment]
@gshomo & @SerenaCaplins, could you please review this piece by EOD tomorrow? Thanks.

**Comment by manasvita** [comment]
@gshomo, & @tketchem14, added verbiage around maintenance. Please check and approve. Thanks.

**Review by SerenaCaplins — APPROVED** [review]
Overall everything looks good to me! Just a few small changes suggested.

File: docs/source/hardware/partitions.md [hunk 1]
```diff
@@ -4,19 +4,47 @@
...
+Group. Core and RAM limits are set per user, across all running jobs
+(not pending). **Keep in mind that the number of running jobs is
+limited by the available resources on the cluster at the time of the
+job submission and may not adhere to the number stated below.**
+
+*job violates accounting/QOS policy*
+
+Sometimes it could happen that, on a shared or a private partition,
+you submitted less jobs than the number of submitted jobs indicated in
+the table, but ended up getting the above message upon doing `squeue
```
[the remark this answers is not in the document]
Comment (manasvita) [reply docs/source/hardware/partitions.md:41]: @SerenaCaplins, thanks for the suggestion. I rephrased that entire para. Please take a look.

File: docs/source/hardware/partitions.md [hunk 2]
```diff
@@ -4,19 +4,47 @@
...
-In the following table, the Running Jobs Per User/Per Research Group. Core and RAM limits are set per user, across all running jobs (not pending). **Keep in mind that the number of running jobs is limited by the available resources on the cluster at the time of the job submission and may not adhere to the number stated below.**
+In the following table, the Running Jobs Per User/Per Research
+Group. Core and RAM limits are set per user, across all running jobs
+(not pending). **Keep in mind that the number of running jobs is
+limited by the available resources on the cluster at the time of the
+job submission and may not adhere to the number stated below.**
+
+*job violates accounting/QOS policy*
+
+Sometimes it could happen that, on a shared or a private partition,
```
[the remark this answers is not in the document]
Comment (manasvita) [reply docs/source/hardware/partitions.md:39]: Rephrased it.

**Comment by manasvita** [comment]
@SerenaCaplins, thanks for providing your feedback. I've incorporated all the suggestions. Take a look.

**Pull request merged**